src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,53 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Calculator {
+
+    int resultMethod(String cal, String method) {
+        int splitNum, splitNum2;
+        int result = 0;
+        String reg = "[+*/-]";
+        String[] split = cal.split(reg);
+        splitNum = Integer.parseInt(split[0]);
+        splitNum2 = Integer.parseInt(split[1]);
+        if (method.equals("+")) {
+            result = splitNum + splitNum2;
+        } else if (method.equals("-")) {
+            result = splitNum - splitNum2;
+        } else if (method.equals("*")) {
+            result = splitNum * splitNum2;
+        } else if (method.equals("/")) {
+            result = splitNum / splitNum2;
+        }
+        return result;
+    }
+
+    String requestCal(String str) {
+        AtomicInteger i = new AtomicInteger(1);
+
+        String reg = "[+*/-]";
+        String reg_num = "[0-9]+";
+
+        ArrayList<String> ch = new ArrayList<>(Arrays.asList(str.replaceAll(" ", "").split(reg_num)));
+        ArrayList<String> arr = new ArrayList<>(Arrays.asList(str.replaceAll(" ", "").split(reg)));
```

Comment:
```suggestion
        List<String> ch = new ArrayList<>(Arrays.asList(str.replaceAll(" ", "").split(reg_num)));
        List<String> arr = new ArrayList<>(Arrays.asList(str.replaceAll(" ", "").split(reg)));
```

List로 선언해 다양성을 가져보는 것은 어떨까요?

그리고 
`List<String> arr`을  `List<Operator> arr`로 만들어 값을 포장해보는 것은 어떨까요?